src/components/Product.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import { prodData } from '../globalTypes';
+import Button from './Button';
+import { propsProd } from '../../src/globalTypes';
+import { Link } from 'react-router-dom';
+
+const Product = (props: propsProd) => {
+    return (
+        <div
+            className='product-item'
+            style={{
+                backgroundImage: `url(${props.prod.images[0]})`,
+                backgroundSize: 'cover',
+            }}
+        >
+            <div className='desc-text'>
+                <div className='item-title'>
+                    {props.prod.title}
+                </div>
+                <div className='item-info'>
+                    <div className='category'>
+                        {`Category: ${props.prod.category}`}
+                    </div>
+                    <div className='brand'>
```

Comment:
From my point of view it possible to create a component. Which will return receive className and string with title. And return div.
In that case there will be only 6 function call. Or even greater variant - array with data. Each item is an object with className + string. And we can just iterate throw it via .map and return divs.